### PR TITLE
Added table to migration, updated screenshots in the Subscriber Source topic

### PR DESCRIPTION
**Pull request by hannanelson**
I didn't have the description of `fetchSize` in polling subscriber-types, so I left it as it was in 2.x. Let me know if you want a PDF to see the screenshots in context.

### CODE REVIEW

**Review by podviaznikov — APPROVED** [review]
looks good to me

File: modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-new-3x-major.adoc [hunk 1]
```diff
@@ -184,6 +184,26 @@ In version 3.x of this connector, the polling mode of the Subscriber source is s
...
 ----
 
+This table lists the 2.x `polling` parameters and their 3.x equivalents.
+
+
+[%header,cols="20a,30a,20a,30a"]
+|===
+| Version 2.x Parameter |Description| Version 3.x Equivalent |Description
+|`fetchSize`| Declared at the `config` level, controls the size of the local message buffer, which is calculated as three times the `fetchSize` value.
+|`fetchSize`| Declared at the `Subscriber` level, controls the size of the local message buffer, which is calculated as three times the `fetchSize` value.
```
Comment (alegmarra) [inline modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-new-3x-major.adoc]: Please change to:
"Sets the maximum amount of messages to be fetched on each polling execution, in range from 1 to 10 messages"

This is based on the Javadoc of the parameter: 
https://github.com/mulesoft/anypoint-mq-connector/blob/develop-4.x/anypoint-mq-extension/src/main/java/com/mulesoft/extension/mq/api/source/PollingTypeSubscriberFactory.java#L57

File: modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-new-3x-major.adoc [hunk 2]
```diff
@@ -158,7 +158,7 @@ In version 3.x of this connector, the polling mode of the Subscriber source is s
 <anypoint-mq:subscriber config-ref="Anypoint_MQ_Config"
   destination="queue">
    <anypoint-mq:subscriber-type >
-      <anypoint-mq:polling >
+      <anypoint-mq:polling fetchSize="20">
```
Comment (alegmarra) [inline modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-new-3x-major.adoc]: invalid value, the range is from 1 to 10. The default is 10.

File: modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-new-3x-major.adoc [hunk 3]
```diff
@@ -174,7 +174,7 @@ In version 3.x of this connector, the polling mode of the Subscriber source is s
 <anypoint-mq:subscriber destination="queue"
     config-ref="Anypoint_MQ_Config">
     <anypoint-mq:subscriber-type>
-        <anypoint-mq:polling>
+        <anypoint-mq:polling fetchSize="20">
```
Comment (alegmarra) [inline modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-new-3x-major.adoc]: invalid value, the range is from 1 to 10. The default is 10.

**Review by alegmarra — CHANGES_REQUESTED** [review]

**Review by alegmarra — APPROVED** [review]

File: modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-listener.adoc [hunk 4]
```diff
@@ -9,9 +9,12 @@ The Subscriber source in the Anypoint MQ connector enables the app to listen for
 
 By default, the Subscriber source works in continuous listening mode, providing maximum throughput by consuming messages as soon as they arrive in the queue. This is called the `prefetch` subscriber-type configuration. When using the `prefetch` mode, the Subscriber source attempts to keep a local buffer of messages full, making them available to be dispatched to the Mule flow as soon as the app can accept them.
 
-For `subscriber-type`, you configure the `maxLocalMessages` parameter to specify the target size for a full buffer, which controls how quickly messages are taken from the queue.
+.The arrow shows the value of the *Subscriber type* field set to *Prefetch* in the *Subscriber* properties window.
+image::anypoint-mq/3.x/amq-3x-subscriber-prefetch.png[Prefetch subscriber type]
+
+
+For the `prefetch` subscriber-type, you configure the `maxLocalMessages` parameter to specify the target size for a full buffer, which controls how quickly messages are taken from the queue.
```
Comment (kbostrom) [inline modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-listener.adoc]: Why is "subscriber-type" hyphenated here?
I see that it is earlier, in "`prefetch` subscriber-type configuration," which makes sense as a compound adjective, but I also see that the alt text uses "subscriber type"?

File: modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-listener.adoc [hunk 5]
```diff
@@ -42,11 +45,11 @@ If your app requires predictable and controlled message consumption rather than
 Messages aren't consumed immediately. Instead, they remain in the queue until the next poll of the Subscriber source is triggered. The queue contains up to 10 messages to be retrieved. For example, if your queue has 15 messages, one poll consumes the first 10 messages and the remaining 5 messages wait for the next poll.
 
 
-// *IS THIS SCREENSHOT CORRECT?*
+The most basic `polling` subscriber-type declaration looks like this:
```
Comment (kbostrom) [inline modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-listener.adoc:48]: hyphen makes sense to me here

File: modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-listener.adoc [hunk 6]
```diff
@@ -60,7 +63,8 @@ The resulting `MuleMessage` has:
 * The message body as its payload.
 * The message metadata in the message attributes.
 
-image::anypoint-mq/3.x/amq-3x-subscriber-output.png[Anypoint MQ Subscriber Output Message]
+.The screenshot shows (*1*) the *Payload* and (*2*) the *Attributes* of the message in the *Output* tab.
```
Comment (kbostrom) [inline modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-listener.adoc]:
```suggestion
.The screenshot shows (*1*) *Payload* and (*2*) *Attributes* of the message in the *Output* tab.
```

File: modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-new-3x-major.adoc [hunk 7]
```diff
@@ -184,6 +184,27 @@ In version 3.x of this connector, the polling mode of the Subscriber source is s
...
+This table lists the 2.x `polling` parameters and their 3.x equivalents.
+
+
+[%header,cols="20a,30a,20a,30a"]
+|===
+| Version 2.x Parameter |Description| Version 3.x Equivalent |Description
+|`fetchSize`| Declared at the `config` level, controls the size of the local message buffer, which is calculated as three times the `fetchSize` value.
+|`fetchSize`| Declared at the `Subscriber` level, sets the maximum number (1-10) of messages to fetch on each polling execution.
+The default is 10.
+
```
Comment (kbostrom) [inline modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-new-3x-major.adoc:196]: Just checking: both parameters named the same, right?

File: modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-new-3x-major.adoc [hunk 8]
```diff
@@ -184,6 +184,27 @@ In version 3.x of this connector, the polling mode of the Subscriber source is s
...
+|`fetchTimeout`| Controls the time waiting for a response from the service.
```
Comment (kbostrom) [inline modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-new-3x-major.adoc]:
```suggestion
|`fetchTimeout`| Limits the wait time for a response from the service.
```

File: modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-new-3x-major.adoc [hunk 9]
```diff
@@ -184,6 +184,27 @@ In version 3.x of this connector, the polling mode of the Subscriber source is s
...
+|Deprecated| This parameter is no longer available. The connector always uses the maximum long-polling value.
+
+|`frequency`| Controls when to recheck for messages after the queue is empty.
+|Deprecated | This parameter is no longer available. The connector handles the frequency internally, fixed at 1 second.
```
Comment (kbostrom) [inline modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-new-3x-major.adoc]:
```suggestion
|Deprecated | This parameter is no longer available. The connector uses a fixed frequency of 1 second.
```

**Review by kbostrom — APPROVED** [review]
Edited

File: modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-new-3x-major.adoc (hunk 7 above)
Comment (hannanelson) [reply modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-new-3x-major.adoc:196]: Yes.

**Pull request merged**